tests/Unit/Form/FormBuilderTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/** @noinspection PhpUndefinedMethodInspection, PhpUnhandledExceptionInspection,StaticClosureCanBeUsedInspection */
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Form;
+
+use MyParcelNL\Pdk\Base\Factory\PdkFactory;
+use MyParcelNL\Pdk\Form\Collection\InputOptionsCollection;
+use MyParcelNL\Pdk\Form\Helpers\FormBuilder;
+use MyParcelNL\Pdk\Form\Model\Input\CheckboxInput;
+use MyParcelNL\Pdk\Form\Model\Input\HiddenInput;
+use MyParcelNL\Pdk\Form\Model\Input\RadioButtonInput;
+use MyParcelNL\Pdk\Form\Model\Input\Select\CountrySelect;
+use MyParcelNL\Pdk\Form\Model\Input\Select\DropOffDaySelect;
+use MyParcelNL\Pdk\Form\Model\Input\SelectInput;
+use MyParcelNL\Pdk\Form\Model\Input\TextInput;
+use MyParcelNL\Pdk\Form\Model\Input\ToggleInput;
+use MyParcelNL\Pdk\Settings\Model\View\CarrierSettingsView;
+use MyParcelNL\Pdk\Settings\Model\View\CheckoutSettingsView;
+use MyParcelNL\Pdk\Settings\Model\View\CustomsSettingsView;
+use MyParcelNL\Pdk\Settings\Model\View\GeneralSettingsView;
+use MyParcelNL\Pdk\Settings\Model\View\LabelSettingsView;
+use MyParcelNL\Pdk\Settings\Model\View\OrderSettingsView;
+use MyParcelNL\Pdk\Tests\Bootstrap\MockPdkConfig;
+
+beforeEach(function () {
+    PdkFactory::create(MockPdkConfig::DEFAULT_CONFIG);
+    $this->formBuilder = new FormBuilder();
+});
+
+it('get customs settings view form', function () {
+    $fields = $this->formBuilder->getCustomsSettingsView();
+    expect($fields)
+        ->toBeInstanceOf(CustomsSettingsView::class)
+        ->and($fields->getAttribute('defaultCustomsCode'))
+        ->toBeInstanceOf(TextInput::class)
+        ->and($fields->getAttribute('defaultCountryOrigin'))
+        ->toBeInstanceOf(CountrySelect::class);
+});
+
+it('get label settings view form', function () {
+    $fields = $this->formBuilder->getLabelSettingsView();
+    expect($fields)
+        ->toBeInstanceOf(LabelSettingsView::class);
+});
+
+it('get order settings view form', function () {
+    $fields = $this->formBuilder->getOrderSettingsView();
+    expect($fields)
+        ->toBeInstanceOf(OrderSettingsView::class)
+        ->and($fields->getAttribute('ignoreOrderStatuses'))
+        ->toBeInstanceOf(InputOptionsCollection::class)
+        ->and($fields->getAttribute('orderStatusMail'))
+        ->toBeInstanceOf(ToggleInput::class)
+        ->and($fields->getAttribute('statusOnLabelCreate'))
+        ->toBeInstanceOf(SelectInput::class);
+});
+
+it('get checkout settings view form', function () {
+    $fields = $this->formBuilder->getCheckoutSettingsView();
+    expect($fields)
+        ->toBeInstanceOf(CheckoutSettingsView::class);
+});
+
+it('get general settings view form', function () {
+    $fields = $this->formBuilder->getGeneralSettingsView();
+    expect($fields)
+        ->toBeInstanceOf(GeneralSettingsView::class);
+});
+
+it('get carrier settings view form', function () {
```

Comment:
Al deze tests testen helemaal niks... alleen of je je model wel goed hebt geschreven (of niet) en of er een exception ontstaat bij het initiëren... 

Het heeft nog meer zin om die formulieren te `toArray`'en en dat te vergelijken (met `Arr::dot` voor leesbaarheid). Dan zorg je tenminste dat we nooit onbewust een formulier kunnen wijzigen